src/apps/loan-list/materials/stackable-material/stackable-material.tsx

```diff
@@ -44,7 +44,7 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
         "list-reservation--stacked": additionalMaterials > 0
       })}
       role="button"
-      onClick={() => openLoanDetailsModalHandler()}
+      onClick={() => openDueDateModal && dueDate && openDueDateModal(dueDate)}
```

Comment:
I think it would be nice to lift this up into a function - the exact same line is repeated four times